sphinx/screens-detail/create-tribe/create-tribe/src/main/java/chat/sphinx/create_tribe/ui/CreateTribeViewState.kt

```diff
@@ -1,13 +1,50 @@
 package chat.sphinx.create_tribe.ui
 
+import chat.sphinx.concept_network_query_chat.model.TribeDto
 import io.matthewnelson.concept_views.viewstate.ViewState
 import java.io.File
 
 internal sealed class CreateTribeViewState: ViewState<CreateTribeViewState>() {
     object Idle: CreateTribeViewState()
 
-    object CreatingTribe: CreateTribeViewState()
+    object SavingTribe: CreateTribeViewState()
+
     class TribeImageUpdated(
         val imageFile: File
     ): CreateTribeViewState()
+
+    object LoadingExistingTribe: CreateTribeViewState()
+
+    class ExistingTribe(
+        val name: String,
+        val description: String,
+        val imageUrl: String?,
+        val tags: Array<String>,
+        val priceToJoin: Long,
+        val pricePerMessage: Long,
+        val escrowAmount: Long,
+        val hourToStake: Long = 0,
```

Comment:
overridden arguments should always go at the end of the constructor arguments, although based on the operator function this override isn't necessary.